### ISSUE TITLE
add dplay.es?

### ISSUE THREAD

**Issue by Sirfrancis18** [spaam/svtplay-dl#1285]
Is it possible for you to add the website dplay.es?

**Comment by spaam** [comment]
yes it might be possible. there is some places where we set the url to a specific `program`  in swedish  and `programmer`  in danish/norwegian. i guess i have to google translate that to spanish and see if it might work.

**Comment by Sirfrancis18** [comment]
thank you very much

**Comment by Sirfrancis18** [comment]
> yes it might be possible. there is some places where we set the url to a specific `program` in swedish and `programmer` in danish/norwegian. i guess i have to google translate that to spanish and see if it might work.

If you want, you can give me what needs to be translated and I will compare it with the Spanish website. thanks

**Comment by felisucoibi** [comment]
+1 if you need help translating or oher related stuff to make it work for dplay.es

**Comment by felisucoibi** [comment]
The word to twshow in the url is "series"
https://www.dplay.es/series/fast-n-loud

**Comment by felisucoibi** [comment]
Channels url:
https://www.dplay.es/series-canales-tv

Concrete channels:
we have two in spain, the keyword in the url for each channel is "canales"

https://www.dplay.es/canales/dkiss
https://www.dplay.es/canales/dmax
Concrete twshow url:
twshow key url word "series"
example:
https://www.dplay.es/series/menuda-es-mi-madre

I made a PR with all these but probaby i did it wrong.
